test/middleware.ts

```diff
@@ -1,10 +1,16 @@
+import { UnknownError } from 'http-errors';
 import { ExtendableContext } from 'koa';
 import { Middleware } from 'koa-compose';
 import { AppContext } from '../src/app';
 
 export type NextMiddleware<CustomT extends ExtendableContext = AppContext> =
   (context: CustomT) => Promise<void>;
 
+export const throwingNext = <CustomT extends ExtendableContext = AppContext>
+  (error: UnknownError = new Error()): NextMiddleware<CustomT> => (): never => {
```

Comment:
Changed return type to `never`, and removed `async` (seems to make no difference).

Reply:
Given there's no `await` inside roughly I would expect `async` not to be required, but who knows what kind of type checking is going on